src/pages/Confirmation/Confirmation.js

```diff
@@ -0,0 +1,128 @@
+import { withFirebase } from "utils/Firebase";
+import { compose } from "recompose";
+import { Row, Col, Container } from "react-bootstrap";
+import { withRouter, Redirect } from "react-router-dom";
+import { ADMIN_ACCOUNT } from "utils/constants.js";
+import { Button } from "reactstrap";
+import "./ConfirmationStyle.scss";
+import CardsDisplay from "../../components/CardsDisplay/CardsDisplay";
+import ConfirmationCard from "../../components/ConfirmationCard/ConfirmationCard";
+
+function Confirmation(props) {
+  const { firebase } = props;
+  const lesson = props?.location.state;
+  const words = lesson.selectedWords;
+  const deploymentAccountIds = lesson.deploymentIds;
+  const lessonType = lesson.lesson;
+  const wordGroup = lesson.group;
+  const dates = lesson.dueDates;
+  const lessonCards = lesson.cards;
+  const lessonName = lesson.lessonNameValue;
+  const existingId = lesson.id;
+  const [submitted, setSubmitted] = useState(false);
+  const [editRedirect, setEditRedirect] = useState(false);
+  const [handlePhonics, setPhonics] = useState(false);
+  const [handleWords, setWords] = useState(false);
+  const [handleVocab, setVocab] = useState(false);
+
+  useEffect(() => {
+    if (lessonType === "A") setVocab(true);
+    else if (lessonType === "C") setPhonics(true);
+    else if (lessonType === "A3") setWords(true);
+  });
+
+  function pushLesson() {
+    firebase.setCustomLesson(
+      ADMIN_ACCOUNT,
+      deploymentAccountIds,
+      lessonType,
+      wordGroup,
+      words,
+      dates,
+      lessonName,
+      existingId
+    );
+    setSubmitted(true);
+  }
+
+  if (submitted) {
+    return <Redirect to="/" />;
+  }
+
+  if (editRedirect) {
+    return (
+      <Redirect
+        to={{
+          pathname: "/createlesson",
+          state: {
+            existingAssignment: {
+              deploymentAccountIds: deploymentAccountIds,
+              words: words,
+              lessonTemplate: lessonType,
+              wordGroup: wordGroup,
+              dueDates: dates,
+              lessonName: lessonName,
+              id: existingId
+            }
+          }
+        }}
+      />
+    );
+  }
+
+  return (
+    <div>
+      <div className="FlexContainer">
+        <div className="WordDisplayBar">
+          {handlePhonics && (
+            <div className="LessonTypeDisplayIcon">
+              <img src="images/lesson-group/phonics.svg" alt="phonics" />
+              PHONICS
+            </div>
+          )}
+          {handleVocab && (
+            <div className="LessonTypeDisplayIcon">
+              <img src="images/lesson-group/words.svg" alt="words" />
+              VOCAB
+            </div>
+          )}
+          {handleWords && (
+            <div className="LessonTypeDisplayIcon">
+              <img src="images/lesson-group/writing.svg" alt="writing" />
+              WRITING
+            </div>
+          )}
+          <div>
+            <ConfirmationCard
+              lessonDate={"WORDS"}
```

Comment:
why is the lessondate = "WORDS"- what is this line doing

Reply:
uhhhh i repurposed some of prashants code oooops updated it now.
its basically the title of the card and before it was always a date but now its either a title or a date.